<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtxtReader</class>
 <widget class="QWidget" name="qtxtReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="f_toolbar" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="l_encode_name">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        
        <property name="text">
         <string>encode_name</string>
        </property>
        
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      
     
      <item>
       <widget class="QLabel" name="l_word_count">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
       
        <property name="text">
         <string>word_count</string>
        </property>
       </widget>
      </item>
      
      
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="f_menu">
        <property name="text">
         <string>菜单[SEL]</string>
        </property>
       </widget>
      </item>
     </layout>
    
     <zorder>line</zorder>
     <zorder>l_encode_name</zorder>
     <zorder>f_menu</zorder>
     <zorder>l_word_count</zorder>
    </widget>
   </item>
   <item>
    <widget class="MTextEdit" name="f_textedit">
     <property name="autoFormatting">
      <set>QTextEdit::AutoNone</set>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="ReadOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MTextEdit</class>
   <extends>QTextEdit</extends>
   <header>mtextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>f_textedit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
